go-apps/meep-rnis/server/rnis.go

```diff
@@ -307,6 +309,7 @@ func createClient(notifyPath string) (*clientNotif.APIClient, error) {
 func checkForExpiredSubscriptions() {
 
 	nowTime := int(time.Now().Unix())
+	mutex.Lock()
```

Comment:
Same recommendation as above regarding a defer of the mutex.Unlock().

Reply:
defer added everywhere to prevent future error.